cohort-engine/src/main/java/com/ibm/cohort/engine/ClasspathLibrarySourceProvider.java

```diff
@@ -21,14 +21,13 @@ public ClasspathLibrarySourceProvider(List<String> libraryResources,
 			FilenameToVersionedIdentifierStrategy idStrategy) {
 		for (String resource : libraryResources) {
 			VersionedIdentifier vid = idStrategy.filenameToVersionedIdentifier(resource);
-			Map<LibraryFormat, InputStream> formats = sources.computeIfAbsent(vid, key -> {
-				return new HashMap<LibraryFormat, InputStream>();
-			});
+			Map<LibraryFormat, InputStream> formats = sources.computeIfAbsent(vid, key -> new HashMap<>());
 			InputStream is = ClassLoader.getSystemResourceAsStream(resource);
 			if( is == null ) {
 				throw new IllegalArgumentException( resource );
 			}
 			formats.put(LibraryFormat.forString(resource), is);
 		}
+		FhirHelperBackupProvider.addClasspathFhirHelpers(sources);
```

Comment:
Since it seems like all source providers want the correct `FHIRHelpers` instance loaded, would it make more sense to move this logic to the `MultiFormatLibrarySourceProvider` constructor?  That should allow us to move the logic from `FhirHelperBackupProvider` to `MultiFormatLibrarySourceProvider` as well.

We've already long gone down the path of super intelligent constructors, what's the harm in adding another in the base class?